gallery/src/PageHeader/HostedVersionsSelect.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useEffect, ChangeEventHandler, useState } from 'react';
+import { NxFormSelect } from '@sonatype/react-shared-components';
+
+import packageJson from '../../package.json';
+
+import './HostedVersionsSelect.scss';
+
+const CURRENT_VERSION = packageJson.version;
+const HOSTED_VERSIONS_JSON_URL = 'https://gallery.sonatype.dev/versions/latest/hosted-versions.json';
+
+const getHostedVersion = () => fetch(HOSTED_VERSIONS_JSON_URL, {
+  headers: {
+    'Content-Type': 'application/json',
```

Comment:
I don't think this header is needed.  As a `GET` it has no request body and so no need for a Content-Type

Reply:
> I don't think this header is needed. As a `GET` it has no request body and so no need for a Content-Type

Updated here: 452ee649b08bf6659d64b8b0410767d0dc26ef30